test/features/deploy_quarkus_service.feature

```diff
@@ -131,9 +131,9 @@ Feature: Deploy quarkus service
       | config | persistence | enabled  |
       | config | events      | enabled  |
     And Kogito application "process-quarkus-example" has 1 pods running within <minutes> minutes
-    And HTTP GET request on service "process-quarkus-example" with path "orders" is successful within 3 minutes
+    And Service "process-quarkus-example" contains 0 instances of process with name "orders" within 3 minutes
 
-    When HTTP POST request on service "process-quarkus-example" with path "orders" and body:
+    When Start "orders" process on service "process-quarkus-example" with body:
```

Comment:
could be replaced by:
`When Start "orders" process on service "process-quarkus-example" within 3 minutes with body:`

Reply:
Done